easybuild/easyconfigs/b/BRHCLUS/BRHCLUS-4.7.2-foss-2021a.eb

```
@@ -0,0 +1,46 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'CMakeMakeCp'
+
+name = 'BRHCLUS'
+version = '4.7.2'
+
+homepage = "https://www.orthodb.org/"
+description = """OrthoDB standalone pipeline for delineation of orthologs"""
```

```suggestion
description = """This package implements a rule based clustering. It takes as input a file with best-reciprocal-hits
 and finds clusters."""
```
Should BRHCLUS have a different description to ORTHOLOGER? The suggestion is from the BRHCLUS README.